contracts/validator_manager.v.py

```diff
@@ -234,6 +248,46 @@ def release_notary() -> bool:
     return True
 
 
+# Given notary, shard_id and index, return True/False to indicate if the
+# notary has been chosen as a member of committee in the specified shard and period
+@public
+@constant
+def is_member_of_committee(
```

Comment:
Just thought that why we have `is_member_of_committee(shard_id, notary, index) -> bool` instead of  `get_member_of_committee(shard_id, index) -> address`?
Seems we can pass less arguments if we use `get_member_of_committee` instead.

Reply:
Yes, this makes sense.